src/ci/bin/test_slurm.inc.sh

```diff
@@ -44,6 +44,7 @@ ReturnToService=1
 SelectType=select/cons_res
 SelectTypeParameters=CR_CPU
 SlurmctldDebug=3
+ClusterName=SlurmClusterForCentaurTests
```

Comment:
Tol: Leave this list alphabetized so it doesn't end up in a random order, and use `localhost` the same as the other names above like the machine and node.

Or, adjust those names to make them consistent with your addition.